fem/fe_coll.hpp

```diff
@@ -936,10 +937,11 @@ class Const3DFECollection : public FiniteElementCollection
 private:
    const P0TetFiniteElement TetrahedronFE;
    const P0HexFiniteElement ParallelepipedFE;
-   const L2_WedgeElement WedgeFE;
+   const P0WdgFiniteElement WedgeFE;
+   const P0PyrFiniteElement PyramidFE;
```

Comment:
Not a huge fan of the `Wdg` and `Pyr` abbreviations, but I guess they fit with `Tet` and `Hex`, and I can't think of anything better...

Reply:
Agreed